app/services/decline_qts.rb

```diff
@@ -13,6 +13,7 @@ def call
 
     ActiveRecord::Base.transaction do
       application_form.update!(declined_at: Time.zone.now)
+      create_timeline_event
```

Comment:
Could we create the timeline event after `ApplicationFormStatusUpdater.call`? This would mean that the application declined event appears as the most recent event and the stage/action required by changed timeline events appear below it.